src/main/java/org/sagebionetworks/web/client/widget/accessrequirements/ManagedACTAccessRequirementWidgetViewImpl.java

```diff
@@ -344,4 +354,37 @@ public void setAccessRequirementName(String description) {
       accessRequirementDescription.removeStyleName("boldText");
     }
   }
+
+  @Override
+  public void showRequestAccessModal(
+    ManagedACTAccessRequirement accessRequirement
+  ) {
+    try {
+      JSONObjectAdapter arAsJson = accessRequirement.writeToJSONObject(
```

Comment:
It's funny how we go from one strongly typed environment to another via json, it's a lot like a client/server interface